pkg/email/system_intake_admin_upload_doc_test.go

```diff
@@ -0,0 +1,84 @@
+package email
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/google/uuid"
+
+	"github.com/cms-enterprise/easi-app/pkg/models"
+)
+
+func (s *EmailTestSuite) TestSendSystemIntakeAdminUploadDocEmail() {
+	ms := mockSender{}
+
+	ctx := context.Background()
+	intakeID := uuid.MustParse("24dd7736-e4c2-4f67-8844-51187de49069")
+	requestName := "Hotdog/Not Hotdog Program"
+	requester := "Dr Fishopolis"
+	requestComponent := "DOC"
+
+	requestLink := fmt.Sprintf(
+		"%s://%s/governance-task-list/%s",
+		s.config.URLScheme,
+		s.config.URLHost,
+		intakeID.String(),
+	)
+
+	recipient := models.NewEmailAddress("fake@fake.com")
+
+	getExpectedEmail := func() string {
+		return fmt.Sprintf(`
+		<h1 class="header-title">EASi</h1>
+		<p class="header-subtitle">Easy Access to System Information</p>
+
+		<p>New documents have been added to GRB review for %[1]s. You can view the new documents using the link below.</p>
```

Comment:
Wow, I can't believe I didn't know about [this](https://stackoverflow.com/questions/74259690/how-to-index-parameters-when-formatting-a-string-in-go). Didn't find it in go by example or the fmt pkg documentation either.

Reply:
yeah they help a ton for readability